config/locales/en.yml

```
@@ -530,8 +525,9 @@ en:
       external_users: External Users
       finishing_rate: Finishing Rate
     submit:
-      failure: An error occurred while transmitting your score. Please try again later.
-      too_late: Your submission was saved successfully but was received after the deadline passed.
+      failure: The scoring was performed successfully but an error occurred while transmitting your score. Please try again later.
+      too_late: Your submission was saved successfully but was received after the deadline passed so that only %{score_sent} points were transmitted.
+      full_score: Congratulations! You have achieved the highest possible score for this exercise. Click <a href="%{url}" class="alert-link">here</a> to complete the exercise.
```

I would suggest highlighting the link a bit more, so that it's clearer to find. 

```suggestion
      full_score: Congratulations! You have achieved the highest possible score for this exercise. <a href="%{url}" class="alert-link">Please click here to finish the exercise now.</a>
```